src/k8s-engine/wsl.ts

```diff
@@ -873,6 +875,41 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     const nerdctlPath = await this.wslify(windowsNerdctlPath);
 
     await this.runInstallScript(INSTALL_WSL_HELPERS_SCRIPT, 'install-wsl-helpers', nerdctlPath);
+    await this.installCredentialHelper();
```

Comment:
Could this not run in parallel (in `start()`)?

Reply:
Sure.